src/h3lib/include/h3api.h.in

```diff
@@ -422,7 +463,8 @@ DECLSPEC H3Index H3_EXPORT(h3ToParent)(H3Index h, int parentRes);
  */
 /** @brief determines the maximum number of children (or grandchildren, etc)
  * that could be returned for the given hexagon */
-DECLSPEC int H3_EXPORT(maxH3ToChildrenSize)(H3Index h, int childRes);
+DECLSPEC uint64_t H3_EXPORT(maxH3ToChildrenSize)(H3Index h,
```

Comment:
Please note this is not a backwards compatible change

Reply:
Should I document that somewhere, or do we just want to bundle this up with other 4.0 backward incompatible changes?

Reply:
I can add a note to the changelog

Reply:
This would have to be a 4.0 thing, I think. It has a very minimal impact on the users to make this change, and probably slightly slower than before? So I don't think it should go in the 3.x releases.